default.nix

```diff
@@ -32,19 +32,18 @@ let
     groundhog = repos.groundhog + /groundhog;
     groundhog-postgresql = repos.groundhog + /groundhog-postgresql;
     groundhog-th = repos.groundhog + /groundhog-th;
-    monoidal-containers = repos.monoidal-containers;
     postgresql-lo-stream = repos.postgresql-lo-stream;
     # Newer versions than those in reflex-platform
     gargoyle = repos.gargoyle + /gargoyle;
     gargoyle-postgresql = repos.gargoyle + /gargoyle-postgresql;
     gargoyle-postgresql-connect = repos.gargoyle + /gargoyle-postgresql-connect;
     gargoyle-postgresql-nix = repos.gargoyle + /gargoyle-postgresql-nix;
+    libsystemd-journal = repos.libsystemd-journal; # TODO: Remove override once nixpkgs is bumped to include 1.4.4.
```

Comment:
We can also bump `reflex-platform`'s all cabal hashes.

Reply:
That wouldn't be enough as this requires some custom nixpkgs config that I copied from upstream. So we might as well just do it this way...